lib/src/Base/Func/HyperbolicAnisotropicEnumerateFunction.cxx

```diff
@@ -275,6 +276,18 @@ void HyperbolicAnisotropicEnumerateFunction::setUpperBound(const Indices & upper
   initialize();
 }
 
+/* The marginal enumerate function */
+EnumerateFunction HyperbolicAnisotropicEnumerateFunction::getMarginal(const Indices & indices) const
+{
+  const UnsignedInteger inputDimension = getDimension();
+  indices.check(inputDimension);
```

Comment:
check just returns a boolean, you have to throw if false

Reply:
Very true: fixed at a3e4351f503ea9976732960cfb0f4917c6b05d94.